docs/User-Experience/Research-Syntheses/2023, Spring - Testing CSV & Excel-based error reports.md

```diff
@@ -0,0 +1,209 @@
+    - **Impact** After discussions with ACF regarding upcoming revisions to the instructions "Item number" does reflect the preferred terminology going forward. This will be captured in [#2527](https://github.com/raft-tech/TANF-app/issues/2527) and additionally justify future knowledge center content to address any resulting confusion for TDP users. 
+
+**Overall the feedback on the post data file submission was positive and encouraging to continue developing the email communication notice and Error Report.**
+
+> “This [Excel file] is helpful too [...] after further review, I’m able to understand the spreadsheet more. I’m excited for this project!" - Tribe
+
+> "I think this excel sheet is more user friendly than the [legacy system report] table. I think this has more information than compared to what I get now" - State
+
+> "[...]I think this is perfect. Once I knew what I was looking at it’s very easy to follow." - Tribe
+
+> “I’m excited about [having this in] an online portal. One of the things that was an issue prior was they would ask for proof that we submitted and we didn’t have it.” - Tribe
+
+> "I like the words and not just the [item] numbers. This saves me a step." - State
+___
+
+
+
+### Who we talked to
+
+We recruited based primarily on the following criteria:
+* Participants from past error message research sessions
+* TANF programs which had previously encountered errors on their Section 1 data files
+* Programs and past participants who had expressed specific interest in participating in research sessions
+
+Research sessions were conducted with:
+* 5 States
+* 4 Tribes
+___
+
+### What we did
+
+Our research was conducted in order to:
+
+* Test the usability of an Excel-based error report located in Submission History to evaluate whether participants would be successful in downloading, opening, and ultimately acting on the reports. 
+* Explore optimal column structures and data needed to help guide error correction and cut down on the time required to understand the report itself.
+
+___
+
+### What we tested
+
+During these sessions, we wanted to simulate the future experience of the error reports with our participants. We asked participants to:
+
+* Speak about their expectations at each step of the process including the error report email, the submission history page (including acceptance status and the link to the error report), and the downloadable error report prototype itself. 
+* Open the error report prototype once they downloaded it. 
+* Identify the meaning in their own words of key columns.
+* Rank the columns in order of their priority in regard to error correction.
+* Use the Error Report as if they were using it to resolve the errors listed and expand on how it might fit into their current workflows.
+
+**Supporting Documentation:**
+- [A variation of our HTML/XLSX prototype](https://reitermb.github.io/HTML-Table-example/xlsx-login)
+___
+
+### What we learned
+
+Jump to: 
```

Comment:
> these links dont work for me. might just be hackmd?

Yep! Hack.md and GitHub use different anchor link conventions. These should all be formatted to work on GitHub. (Preview available here: https://github.com/raft-tech/TANF-app/blob/synthesis-spring-2023/docs/User-Experience/Research-Syntheses/2023%2C%20Spring%20-%20Testing%20CSV%20%26%20Excel-based%20error%20reports.md#What-we-learned)